zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -31,10 +31,12 @@ def handle_message(self, message, bot_handler, state_handler):
         bot_handler.send_reply(message, bot_response)
 
     def get_bot_wiki_response(self, message, bot_handler):
-        help_text = 'Please enter your message after @mention-bot'
+        help_text = 'Please enter your message after @{1}{0}{1}'
         query = message['content']
         if query == '':
-            return help_text
+            name = bot_handler.bot_identity().name
+            pad = '' if name.find(' ') < 0 else '**'
```

Comment:
Right; this comes in to my first and second bullet points in the description. There are various options. Bot names are known in the bot_handler, so this could be handled at that level too, with the aforementioned `quoted_name` providing star-quotes if necessary.

I'm as much interested in the general concept of this PR and whether something like it might be considered, as much as the smaller details.